src/client/eppo-client.spec.ts

```diff
@@ -542,3 +546,133 @@ describe('EppoClient E2E test', () => {
     });
   });
 });
+
+describe(' EppoClient getAssignment From Obfuscated RAC', () => {
```

Comment:
Made a separate suite for the obfuscated RAC because it was too much trouble to switch out the mocked endpoint to a different file and re-initialize the store